packages/components/src/button/test/index.js

```diff
@@ -75,6 +75,29 @@ describe( 'Button', () => {
 			expect( screen.getByRole( 'button' ) ).toHaveClass( 'is-pressed' );
 		} );
 
+		it( 'should render a button element with has-text when children are passed', async () => {
```

Comment:
Added necessary variations to test for the presence of has-text.

Reply:
Added by eaa2dd1

Reply:
I think we need to `.focus()` the button before the assertion so the `.components-tooltip` gets injected as a child.

Reply:
Added by 02ee824